### DISCUSSION TITLE
How to disable Triton warnings?

### DISCUSSION THREAD

**Discussion by JosemyL95** [openai/whisper#1954] [General]
Hi to everyone,
Based on this discussion: https://github.com/openai/whisper/discussions/1283

I realized that the triton warning appears when u don't have the right OS distribution. I'm doing transcriptions (with word_timestamps clause) on Windows 10 and the warning appears when I try to execute a transcription.

Do you know how can I disable ONLY the triton warnings?

Thanks!

**Comment by mjl-ui** [comment]
Hello, how did you solve this issue?
